Testnet release plan.md

```diff
@@ -0,0 +1,55 @@
+# Testnet versions
+
+This is the large task breakdown I propose we add to the next versions of testnet.
+There will be bug fixes and UX improvements as we discover them, which will be treated as high priority.
+
+Note that there is no protocol work listed here. These are the tasks that are orthogonal to the protocol.
+
+
+## V2
+1. Events subscription
+2. Connect to goerli
```

Comment:
I think this means "connect to a L1 testnet", since there's no point in connecting to Goerli per se.